src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -36,6 +36,11 @@
   'resize_method': 'turicreate',
 }
 
+def uniform_num(random_num, min_value, max_value):
+    #return a uniform distribution number between min_value and max_value
```

Comment:
major nit: PEP8 says you should have a space between the `#` and the start of the comment.

Reply:
Good!